downstream/modules/builder/ref-scenario-update-hub-ca-cert.adoc

```diff
@@ -0,0 +1,35 @@
+[id="ref-scenario-update-hub-ca-cert"]
+
+= Updating the {HubName} CA certificate
+
+
+[role="_abstract"]
+Use this example to customize the default definition file to include a CA certificate to the `additional-build-files` section, move the file to the appropriate directory and, finally, run the command to update the dynamic configuration of CA certificates to allow the system to trust this CA certificate.
+
+.Prerequisites
+* A custom CA certificate, for example `rootCA.crt`.
+
+[NOTE]
+====
+Customizing the CA certificate using `prepend_base` means that the resulting CA configuration appears in all other build stages and the final image, because all other build stages inherit from the base image.
```

Comment:
Just a nit (using comma before because; you can disregard, if you prefer)
```suggestion
Customizing the CA certificate using `prepend_base` means that the resulting CA configuration appears in all other build stages and the final image because all other build stages inherit from the base image.
```